teps/0047-finallytask-execution-post-timeout.md

```diff
@@ -0,0 +1,290 @@
+---
+status: proposed
+title: Finally tasks execution post pipelinerun timeout
+creation-date: '2021-01-26'
+last-updated: '2021-01-26'
+authors:
+- '@souleb'
+---
+
+# TEP-0047: Finally tasks execution post pipelinerun timeout
+---
+
+<!-- toc -->
+- [# TEP-0047: Finally tasks execution post pipelinerun timeout](#-tep-0047-finally-tasks-execution-post-pipelinerun-timeout)
```

Comment:
NIT: I don't think this is needed, we can delete this 🤔

Reply:
you can use [`hack/update-toc.sh`](https://github.com/tektoncd/community/blob/main/hack/update-toc.sh) to autofix table of contents